gulpfile.js

```diff
@@ -86,6 +87,15 @@ gulp.task('build-css', css.bind(null, false));
 
 gulp.task('optimize-css', css.bind(null, true));
 
+gulp.task('jscs', function() {
+    gutil.log("JSCS the files...." + '\n');
+    var lintFilter = filter(["**", "!config.js", "!map/us-counties.js", "!map/us-states.js", "!map/countries.js"]);
```

Comment:
I am assuming there will be different files in them at a later point. (I am unsure of it, right now, so for now, should I move them out?)